src/nu/validator/checker/XmlPiChecker.java

```diff
@@ -1377,7 +1377,7 @@ private void attributeNameComplete() throws SAXException {
     private void addAttributeWithValue() throws SAXException {
         if (attributeName != null) {
             String value = longStrBufToString();
-            if (value.indexOf("<") != -1) {
+            if (value.indexOf('<') != -1) {
```

Comment:
> This is just a performance improvement - probably a very small one, but it's no less readable.

Sure yeah it should rightly have been written this way to begin with, so it's nice to get it changed now.